targets/csharp/source/PlayFabSDK/source/Uunit/tests/PlayFabServerApiTest.cs

```diff
@@ -237,6 +240,60 @@ public void ParallelRequest(UUnitTestContext testContext)
                 }
             });
         }
+
+        /// <summary>
+        /// Multiplayer API
+        /// Try to deliberately request a mutliplayer alias that doesn't exist 
+        ///   Verify that response is MutliplayerServerNotFound, not just Not Found
+        /// </summary>
+        [UUnitTest]
+        public async void TestForNotFoundWithImportantInfo(UUnitTestContext testContext)
+        {
+            PlayFabSettings.staticSettings.TitleId = testTitleData.titleId;
+            PlayFabSettings.staticSettings.DeveloperSecretKey = testTitleData.developerSecretKey;
+
+            var eReq = new AuthenticationModels.GetEntityTokenRequest();
+            eReq.Entity = new AuthenticationModels.EntityKey();
+            eReq.Entity.Type = "title";
+            eReq.Entity.Id = testTitleData.titleId;
+
+            var tokenTask = await PlayFabAuthenticationAPI.GetEntityTokenAsync(eReq);
```

Comment:
I migrated all tests to not use static interface anymore.
Please convert this to create and use a local instance.
(this would have avoided your earlier issue as well)